tools/block-generator/runner/run.go

```diff
@@ -112,16 +131,40 @@ func (r *Args) run() error {
 		}
 	}()
 
-	//indexerShutdownFunc, err := startIndexer(dataDir, logfile, r.LogLevel, r.IndexerBinary, algodNet, indexerNet, r.PostgresConnectionString, r.CPUProfilePath)
-	//if err != nil {
-	//	return fmt.Errorf("failed to start indexer: %w", err)
-	//}
-	//defer func() {
-	//	// Shutdown indexer
-	//	if err := indexerShutdownFunc(); err != nil {
-	//		fmt.Printf("Failed to shutdown indexer: %s\n", err)
-	//	}
-	//}()
+	// get conduit config template
+	t, err := template.New("conduit").Parse(conduitConfigTmpl)
+	if err != nil {
+		return fmt.Errorf("unable to parse conduit config template: %w", err)
+	}
+
+	// create config file in the right data directory
+	f, err := os.Create(path.Join(dataDir, "conduit.yml"))
+	if err != nil {
+		return fmt.Errorf("creating conduit.yml: %v", err)
```

Comment:
```suggestion
		return fmt.Errorf("creating conduit.yml: %w", err)
```
nit for consistency with other error wraps here